airflow/exceptions.py

```diff
@@ -44,7 +40,7 @@ class AirflowBadRequest(AirflowException):
     status_code = 400
 
 
-class AirflowNotFoundException(AirflowException, ApiConnexionNotFound):
```

Comment:
I have a feeling that this might break something. Lets see.

Reply:
Same here. Let's see :)